example-setup/clustersetup-env-testing-small.yaml

```diff
@@ -7,11 +7,12 @@ parameters:
   # Used resources can be configures with the following parameters
   number_appservers: 2
   number_dbservers: 1
-  number_servicehosts: 1 
+  number_servicehosts: 1
   flavor_lb: m1.tiny
   flavor_appserver: m1.tiny
   flavor_dbserver: m1.tiny
   flavor_servicehost: m1.tiny
+  mastertokenlengths: 30
```

Comment:
I would prefer something like consul_tokenlength (to state that this token belongs to consul and follow the style of other variables).

Reply:
after discussion consul_mastertoken_length would be the best option